src/components/combobox/combobox.tsx

```diff
@@ -505,7 +509,7 @@ export class Combobox implements LabelableComponent, FormComponent, InteractiveC
       return;
     }
 
-    if (this.allowCustomValues && this.text) {
+    if (this.allowCustomValues && this.text.length) {
```

Comment:
I wonder if we should check for blank spaces before adding a custom chip? Maybe `&& this.text?.trim().length`?